packages/template-retail-react-app/app/components/quantity-picker/index.jsx

```diff
@@ -56,14 +68,39 @@ const QuantityPicker = (props) => {
         })
     })
 
+    // Accessibility improvements:
+    // 1. Allow keyboard focus on the buttons - Chakra overrides values passed to get*ButtonProps()
+    inc.tabIndex = ''
+    dec.tabIndex = ''
+    // 2. Allow Space or Enter key to trigger buttons
+    // Hitting space/enter triggers a "click" event, but the component listens for "mousedown".
+    // We can't reuse the buttons' onMouseDown handler, so instead we hijack the input's onKeyDown.
+    // @ref https://github.com/chakra-ui/chakra-ui/blob/%40chakra-ui/react%402.7.0/packages/components/number-input/src/use-number-input.ts#L333-L334
+    inc.onKeyDown = (evt) => {
+        if (evt.key === ' ' || evt.key === 'Enter') {
+            evt.key = 'ArrowUp'
+            input.onKeyDown(evt)
+        }
+    }
+    dec.onKeyDown = (evt) => {
+        if (evt.key === ' ' || evt.key === 'Enter') {
+            evt.key = 'ArrowDown'
+            input.onKeyDown(evt)
+        }
+    }
+
     return (
         <HStack>
             <Button data-testid="quantity-decrement" {...dec}>
-                -
+                {/* "-" is "hyphen", but we want "minus" */}
+                <FormattedMessage
+                    id="product_view.label.quantity_decrement"
+                    defaultMessage="&minus;"
+                />
             </Button>
             <Input {...input} />
             <Button data-testid="quantity-increment" {...inc}>
-                +
+                <FormattedMessage id="product_view.label.quantity_increment" defaultMessage="+" />
```

Comment:
Why do we need translations for these symbols?

Reply:
> Turn on VoiceOver. Tab into [+] and [-] to see that it says "increment quantity" / "decrement quantity", rather than just "plus" / "minus".

The interaction/functionality works fine but it keeps reading the "Missing translation" error out loud instead of the actual label.